quantumdb-cli/src/main/java/io/quantumdb/cli/xml/XmlAddColumn.java

```diff
@@ -42,7 +42,7 @@ public AddColumn toOperation() {
 		String defaultExpression = column.getDefaultExpression();
 
 		List<Hint> hints = Lists.newArrayList();
-		if (column.isNullable()) {
+		if (!column.isNullable()) {
```

Comment:
This bug happened due to the default value nullable gets if it is not included in the column definition. In the next commit I also changed the default value of nullable:
`column.setNullable(Boolean.TRUE.toString().equals(element.getAttributes().get("nullable")));`
To:
`column.setNullable(!Boolean.FALSE.toString().equals(element.getAttributes().get("nullable")));`
Now when it is not specified, the column is by default nullable instead of not null.